course_contents/19_gui_development_tkinter/lectures/17_adding_permanent_scrollbar/app.py

```diff
@@ -104,53 +108,52 @@ def save_file():
     file_path = filedialog.asksaveasfilename()
 
     try:
-        filename = file_path.split("/")[-1]
-        text_widget = get_text_widget()
-        content = text_widget.get("1.0", "end-1c")
+        filename = os.path.basename(file_path)
+        text_wgt = get_text_widget()
+        content = text_wgt.get('1.0', 'end-1c')
```

Comment:
I don't think this shortening is necessary, it reads better as `text_widget` in my opinion